### ISSUE TITLE
Can it support it running on nginx?

### ISSUE THREAD

**Issue by oyjcmyn** [DanielnetoDotCom/YouPHPTube#64]
Hello~
I hope it can run on nginx.

**Comment by DanielnetoDotCom** [comment]
Hi

I have never installed on nginx, but some one else does https://github.com/DanielnetoDotCom/YouPHPTube/issues/25
It may help you

**Comment by oyjcmyn** [comment]
Thank you very much,I've been running on nginx above. YouPHPtube is really great and i'd love to use it.
But it has been prompted no ffmpeg and exiftool, I have tried a lot of reinstall, still no success.
I installed the site is [http://mv.loveud.com](http://mv.loveud.com)

![02](https://user-images.githubusercontent.com/24839942/28360445-0624ed94-6ca8-11e7-92eb-37661f791993.jpg)
![01](https://user-images.githubusercontent.com/24839942/28360446-065cd0ce-6ca8-11e7-9def-12c771b25a96.jpg)

**Comment by DanielnetoDotCom** [comment]
If you ignore this error and continue install is it working?

**Comment by oyjcmyn** [comment]
I have succeeded in running it on nginx. Only to obtain video metadata (Duration) sometimes can not get to normal.

I have been in the nginx rule configuration, it is better to run on nginx, there are still access to server information (cpu.json) and other rules did not take effect.

I uploaded the file.

Hope that the author can sort out, so youPHPtube can run in more environments.

[nginx.conf.txt](https://github.com/DanielnetoDotCom/YouPHPTube/files/1165394/nginx.conf.txt)
